pontoon/base/tests/test_views.py

```diff
@@ -220,10 +220,7 @@ def test_translation_counts(self):
             'pk': memory_entry.entity.pk,
             'locale': memory_entry.locale.code
         })
-        assert_json(response, [{u'count': 3,
-                     u'quality': 75.0,
-                     u'source': u'abaa',
-                     u'target': u'ccc'}])
+        assert_equal(response.json()[0]['count'], 3)
```

Comment:
I thought about:

```
result = response.json()
assert_equal(len(result), 1)
src_string = result[0].pop('source')
assert_in(src_string, ('aaba', 'abaa'))  # made up, three choices, too
assert_json(result[0], {u'quality': 75.0...
```

Reply:
Updated, let's see what Travis has to say.